### ISSUE TITLE
Inconsistency in Input Pipeline code block for test_dataset

### ISSUE THREAD

**Issue by NathanDotTo** [tensorflow/tensorflow#29581]
## URL(s) with the issue:

https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/generative/pix2pix.ipynb

## Description of issue (what needs changing):

The code block below appears under the heading `Input Pipeline`.
It appears to be an almost, but not quite, semantic copy of the code block above it for the 
`train_dataset`, except that, in this code block for the `test_dataset` we are shuffling the
`train_dataset` again.

```
test_dataset = tf.data.Dataset.list_files(PATH+'test/*.jpg')
# shuffling so that for every epoch a different image is generated
# to predict and display the progress of our model.
train_dataset = train_dataset.shuffle(BUFFER_SIZE)
test_dataset = test_dataset.map(load_image_test)
test_dataset = test_dataset.batch(1)
```

So, I think that this code block should be:

```
test_dataset = tf.data.Dataset.list_files(PATH+'test/*.jpg')
# shuffling so that for every epoch a different image is generated
# to predict and display the progress of our model.
test_dataset = test_dataset.shuffle(BUFFER_SIZE)
test_dataset = test_dataset.map(load_image_test)
test_dataset = test_dataset.batch(1)
```

Also note that the code block for the `train_dataset` has:

```
train_dataset = train_dataset.map(load_image_train,
                                  num_parallel_calls=tf.data.experimental.AUTOTUNE)
```

But the code block for the `test_dataset` only has:
```
test_dataset = test_dataset.map(load_image_test)
```

The `AUTOTUNE ` is noted here, but not explained, so I can't easily comment on what its use,
or lack of, actually means:

https://www.tensorflow.org/api_docs/python/tf/data/experimental#AUTOTUNE

### Submit a pull request?

I can if this is actually incorrect.

**Comment by jvishnuvardhan** [comment]
@NathanDotTo Thanks for finding this. I think this is a typo `train_dataset = train_dataset.shuffle(BUFFER_SIZE)`. 

For more details on AUTOTUNE check this [source](https://www.tensorflow.org/beta/guide/data_performance#pipelining). Thanks!

The other two in your issue

**Comment by amahendrakar** [comment]
@NathanDotTo,
The code snippet has been updated in the [latest version](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/generative/pix2pix.ipynb) of the tutorial.

Cloud you please let us know if this still an issue? Thanks!
